src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <div class="searchBox">
+      <form
+        action="./"
+        method="get"
+        class="bigSearchBar"
+        data-qa="big"
+      >
+        <input
+          type="search"
+          class="searchInput bigSearchInput"
```

Comment:
use one of the inputs as the basic case, and override necessary styles for other cases. You can use the style for big input as the basic and rewrite style for small